ports/python3/portfile.cmake

```
@@ -379,6 +379,17 @@ endif()
 
 if(VCPKG_TARGET_IS_WINDOWS)
   vcpkg_replace_string("${CURRENT_PACKAGES_DIR}/tools/python3/Lib/distutils/command/build_ext.py" "'libs'" "'../../lib'")
+      file(WRITE "${CURRENT_PACKAGES_DIR}/tools/python3/Lib/site-packages/sitecustomize.py"
+[[
+import os
+import sys
+from pathlib import Path
+
+vcpkg_bin_path = Path(sys.prefix + '/../../bin')
+if vcpkg_bin_path.is_dir():
```

```suggestion
if sys.executable.endswith('\\tools\\python3\\python.exe') and Path(sys.prefix + '/../../bin').is_dir():
```
@BillyONeal: Would this be better?

---

there could also be an `else` block which prints a message to remove the file on deployment if it gets accidently deployed.